agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AzureFunctionsLogProcessor.java

```diff
@@ -44,5 +44,8 @@ public void onEmit(Context context, ReadWriteLogRecord logRecord) {
           AiSemanticAttributes.AZ_FN_LIVE_LOGS_SESSION_ID,
           customDimensions.azFunctionLiveLogsSessionId);
     }
+    if (customDimensions.operationName != null) {
+      logRecord.setAttribute(AiSemanticAttributes.OPERATION_NAME, customDimensions.operationName);
+    }
```

Comment:
do we want to stamp this on all telemetry (not only logs)?

Reply:
currently, those attributes from host to worker are only applicable to logs. let's ask function team if needed for all telemetry.